R/namespace.R

```diff
@@ -229,6 +229,14 @@ roxy_tag_parse.roxy_tag_importClassesFrom <- function(x) {
 }
 #' @export
 roxy_tag_ns.roxy_tag_importClassesFrom <- function(x, block, env, import_only = FALSE) {
+  pkg <- x$val[1L]
```

Comment:
I think you meant to do this in `roxy_tag_ns.roxy_tag_importFrom()`

Reply:
:sweat_smile: indeed